portafly/src/i18n/locales/en/audience/account/accounts.yml

```
@@ -0,0 +1,67 @@
+---
+# Accounts > Account overview
+
+page_title: Accounts | Portafly
+page_title_description: Page title of the subsection for Accounts Overview
+
+# Top area
+
+body_title: Account: {{organization_name}}
```

```suggestion
body_title: "Account: {{organization_name}}"
```
Text with format or variables require quotation marks.

---

Thank you :smiley: